src/modules/powerrename/PowerRenameUILib/ExplorerItem.xaml

```diff
@@ -0,0 +1,123 @@
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="4" />
+            <!--  Some spacing in between  -->
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+            <!--  Error tooltip  -->
+        </Grid.ColumnDefinitions>
+
+        <Border
+            Name="HighlightBorder"
+            Grid.ColumnSpan="4"
+            Margin="0,0,0,-1"
+            HorizontalAlignment="Stretch"
+            VerticalAlignment="Stretch"
+            Background="Transparent" />
+
+        <Grid
+            Grid.Column="0"
+            Margin="10,4,0,4"
+            HorizontalAlignment="Left">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition MinWidth="{x:Bind Indentation}" />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+
+
+            <CheckBox
+                Name="{x:Bind IdStr}"
+                Grid.Column="1"
+                Width="28"
+                Height="28"
+                MinWidth="0"
+                AutomationProperties.HelpText="{x:Bind Renamed}"
+                AutomationProperties.Name="{x:Bind Original}"
+                Content=""
+                IsChecked="{x:Bind Checked, Mode=TwoWay}"
+                IsTabStop="True"
+                TabIndex="0"
+                XYFocusKeyboardNavigation="Enabled" />
+            <Image
+                Grid.Column="2"
+                Width="16"
+                Margin="4,0,0,0"
+                HorizontalAlignment="Left"
+                Source="{x:Bind ImagePath}" />
+            <TextBlock
+                Grid.Column="3"
+                Margin="6,0,4,0"
+                VerticalAlignment="Center"
+                FontSize="12"
+                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                Text="{x:Bind Original, Mode=OneWay}"
+                TextTrimming="CharacterEllipsis" />
+        </Grid>
+        <TextBlock
+            Grid.Column="2"
+            Margin="0,0,4,0"
+            VerticalAlignment="Center"
+            FontSize="12"
+            FontWeight="SemiBold"
+            Foreground="{ThemeResource AccentTextFillColorPrimaryBrush}"
+            Text="{x:Bind Renamed, Mode=OneWay}"
+            TextTrimming="CharacterEllipsis" />
+
+        <Border
+            x:Name="ErrorIcon"
+            Grid.Column="3"
+            Width="12"
+            Height="12"
+            CornerRadius="12"
+            Margin="4,0,8,0"
+            Background="{ThemeResource SystemFillColorCriticalBrush}"
+            Visibility="Collapsed">
+            <TextBlock
+                HorizontalAlignment="Center"
+                VerticalAlignment="Center"
+                FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                Foreground="White"
+                FontSize="8"
+                Text="&#xE10A;" />
```

Comment:
Can we use glyph icons with the official brushes and symbols for the error icon in the `muxc:InfoBar`? This would fix the colors and make the `x` centered in the circle.

https://github.com/microsoft/PowerToys/blob/24f4405f2a1a185a3e815a8e898a8a711f4c4997/src/settings-ui/Settings.UI/Views/PowerLauncherPage.xaml#L284

Reply:
@niels9001 E104 is not allowed. You should know this by now. Please remember our agreement.
And please never hard-code a font color.

Reply:
@stefansjfw
Did you saw my review?

Reply:
yes, that's Q for @niels9001 :)

Reply:
@niels9001
Here is a list of all infobar theme resources: https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.infobar?view=winui-2.8#all-theme-resources

Reply:
@stefansjfw and @niels9001 - Using a \<border> to achieve a colored background for an icon? Don't take it the wrong way, but I expect better from you guys (!) (try to find a compliment in there...)

Have a look at https://docs.microsoft.com/nl-nl/windows/apps/design/style/segoe-fluent-icons-font#layering-and-mirroring and let's use icon "[ErrorBadge](https://docs.microsoft.com/nl-nl/windows/apps/design/style/segoe-fluent-icons-font#layering-and-mirroring:~:text=ea39-,ErrorBadge,-ea3a)" (EA39) with a `SystemFillColorCriticalBrush` colored "CircleFill" (EA3B)
_Or maybe icon "Error" (E783) ?_

Reply:
@jay-o-way
I agree with and that's why I wrote it. But fyi the code you suggested won't work. You need a grid containing two icons. One for background and one for the foreground. (See the code in the referenced pr.)

And btw. we can use the background of the error bar as background for the table row. Then we are in line with system colors.

Reply:
> @niels9001
> Here is a list of all infobar theme resources: https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.infobar?view=winui-2.8#all-theme-resources

The InfoBar brushes inherit other brushes... the one we use, so this is fine. See WinUI repo.

Reply:
For the brushes I am okay with what we have. But can we use the X-Glyph-Resource from info bar. The current implementation you did isn't centered on the red background and this looks not good. (See screenshots in PR description.)